app/src/main/res/xml/actions.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version ="1.0" encoding ="utf-8"?>
+<!--  You can learn more about App Actions and how to use them in the documentation:
+https://developers.google.com/assistant/app/overview -->
+
+<actions>
+    <action intentName="actions.intent.OPEN_APP_FEATURE">
+        <fulfillment urlTemplate="https://de.danoeh.antennapod/main{?featureName}">
+            <parameter-mapping
+                intentParameter="feature"
+                urlParameter="featureName" />
+        </fulfillment>
+        <parameter name="feature">
+            <entity-set-reference entitySetId="featureEntitySet" />
+        </parameter>
+    </action>
+
+    <action intentName="actions.intent.GET_THING">
+        <fulfillment urlTemplate="https://de.danoeh.antennapod/search{?query}">
```

Comment:
This can be any URL, right? Is there a best-practice what to use? What do you think about using `https://antennapod.org/deeplink/search{?query}`

Reply:
Yes, it can be any URL. I’ve changed it to your preferred one.